appservice/src/deploy/getDeployFsPath.ts

```diff
@@ -81,3 +84,8 @@ async function appendDeploySubpathSetting(targetPath: string, extensionPrefix: s
 
     return targetPath;
 }
+
+export type IDeployPaths = {
+    originalDeployFsPath: string,
+    effectiveDeployFsPath: string
```

Comment:
nit: Please add comments to each of these describing what they are. Even with the better names they're still a little confusing, plus I think comments are more important in our shared npm packages (that could theoretically be used by other people)